api/common/modelstatus.go

```diff
@@ -103,7 +103,7 @@ func constructModelStatus(model names.ModelTag, owner names.UserTag, r params.Mo
 		}
 		result.Machines[j] = base.Machine{
 			Id:         mm.Id,
-			InstanceId: mm.InstanceId,
+			InstanceId: string(mm.InstanceId),
```

Comment:
There should be no change to InstanceId handling.
But we want to copy DisplayName here right?

Reply:
Fixed.